apps/web/app/(authenticated)/(app)/app/[apiId]/page.tsx

```diff
@@ -61,29 +58,37 @@ export default async function ApiPage(props: { params: { apiId: string } }) {
   }));
 
   return (
-    <div className="grid grid-cols-3 gap-4">
-      <Card className="col-span-1">
+    <div className="grid grid-cols-2 md:grid-cols-3 gap-4">
+      <Card className="">
         <CardHeader>
           <CardTitle>{formatNumber(keys)}</CardTitle>
           <CardDescription>Total Keys</CardDescription>
         </CardHeader>
       </Card>
-      <Card className="col-span-1">
+      <Card className="">
         <CardHeader>
           <CardTitle>
             {formatNumber(active.data.reduce((sum, day) => sum + day.usage, 0))}
           </CardTitle>
           <CardDescription>Active Keys (30 days)</CardDescription>
         </CardHeader>
       </Card>
-      <Card className="col-span-1">
+      <Card className=" col-span-2 md:col-span-1">
         <CardHeader>
           <CardTitle>{formatNumber(usage.data.reduce((sum, day) => sum + day.usage, 0))}</CardTitle>
           <CardDescription>Verifications (30 days)</CardDescription>
         </CardHeader>
       </Card>
-      <Card className="col-span-3">
-        <CardHeader>
+      <Card className="col-span-3 overflow-hidden bg-gradient-to-br from-zinc-50 to-zinc-100 hover:drop-shadow-md dark:from-zinc-950 dark:to-zinc-900 relative">
```

Comment:
I'd prefer it to be clean for now. we're likely going to pivot on the design a lot and introducing gradients and shadows feels like just more work afterwards

Reply:
sure will make it solid it's a barely a gradient but just felt nicer.